frontend/src/concepts/areas/const.ts

```diff
@@ -66,4 +66,8 @@ export const SupportedAreasStateMap: SupportedAreasState = {
     featureFlags: ['disableDistributedWorkloads'],
     requiredComponents: [StackComponent.KUEUE],
```

Comment:
this is intetsting, DW is based on component kueue, what happens if Ray or CFO are set to different managementstate than kueue? and soon coming TO
@lucferbux / @anishasthana

Reply:
This is a great question cc @gitdallas @mturley